utils/libnetwork/Makefile

```diff
@@ -0,0 +1,50 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=libnetwork
+PKG_VERSION:=0.8.0-dev.2
+PKG_RELEASE:=1
+PKG_LICENSE:=Apache-2.0
+PKG_LICENSE_FILES:=LICENSE
+
+GO_PKG:=github.com/docker/libnetwork
+GO_PKG_BUILD_PKG:= \
+  $(GO_PKG)/cmd/proxy \
+  $(GO_PKG)/cmd/dnet
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=git://$(GO_PKG)
```

Comment:
Please consider using `https://` protocol.  Many people are in networks where git tcp port has been blocked.

Reply:
Changed